contentcuration/contentcuration/urls.py

```diff
@@ -84,10 +87,15 @@ def get_redirect_url(self, *args, **kwargs):
     basename="recommendations-interaction",
 )
 router.register(
-    r"communitylibrary_submission",
+    r"communitylibrary-submission",
     CommunityLibrarySubmissionViewSet,
     basename="community-library-submission",
 )
+router.register(
+    r"admin_communitylibrary_submission",
```

Comment:
Either if we use snake case or kebab case for these new routes, I think we should use the same case for both routes.